cmake/coverage.cmake

```diff
@@ -15,12 +15,12 @@ ie_coverage_capture(INFO_FILE "openvino"
 ie_coverage_extract(INPUT "openvino" OUTPUT "inference"
                     PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/inference/*")
 
-ie_coverage_genhtml(INFO_FILE "inference_engine"
+ie_coverage_genhtml(INFO_FILE "inference"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "inference_engine_legacy"
+ie_coverage_extract(INPUT "openvino" OUTPUT "legacy"
                     PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/common/legacy/*")
-ie_coverage_genhtml(INFO_FILE "inference_engine_legacy"
+ie_coverage_genhtml(INFO_FILE "legacy"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
```

Comment:
below we also name incorrect paths like `plugins/hetero_plugin`, names like `inference_engine_transformations`